packages/docusaurus-theme-classic/src/noopModule.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default () => null;
```

Comment:
This is a stub module for `@theme/SearchAlgolia`. Perhaps it can be part of utils in future and shared across themes that need a stub.